.github/workflows/analysis.yml

```diff
@@ -0,0 +1,37 @@
+name: Dart CI
+
+on: [push, pull_request]
+
+jobs:
+  generator_analysis:
+    runs-on: ubuntu-latest
+    container:
+      image:  google/dart:latest
```

Comment:
this seems to be unnecessary, see https://github.com/axel-op/dart_package_analyzer#usage